start.bat

```diff
@@ -0,0 +1 @@
+python test_modules\google_appengine_1.9.27\google_appengine\dev_appserver.py .
```

Comment:
This should probably also go into Gruntfile.js as well, then we can both use it.